src/menus/ListenerMenu.py

```diff
@@ -19,7 +19,7 @@ def autocomplete(self):
         return self._cmd_registry + super().autocomplete()
 
     def get_completions(self, document, complete_event, cmd_line, word_before_cursor):
-        if cmd_line[0] in ['kill', 'info'] and position_util(cmd_line, 2, word_before_cursor):
+        if cmd_line[0] in ['kill', 'info', 'options'] and position_util(cmd_line, 2, word_before_cursor):
```

Comment:
```suggestion
        if cmd_line[0] in ['kill', 'options'] and position_util(cmd_line, 2, word_before_cursor):
```
info doesn't do anything here anymore